tree/ntuple/v7/test/ntuple_evolution.cxx

```diff
@@ -1,62 +1,14 @@
 #include "ntuple_test.hxx"
 
+#define NTUPLE_FORK_MACROS
```

Comment:
Eww, this is ugly. Can we just have all implementations in a header file that we can include into multiple tests?